lib/RawSource.js

```diff
@@ -27,14 +36,22 @@ class RawSource extends Source {
 
 	source() {
 		if (this._value === undefined) {
-			this._value = this._valueAsBuffer.toString("utf-8");
+			const value = internString(this._valueAsBuffer.toString("utf-8"));
+			if (isDualStringBufferCachingEnabled()) {
+				this._value = value;
```

Comment:
```suggestion
			const value = this._valueAsBuffer.toString("utf-8");
			if (isDualStringBufferCachingEnabled()) {
				this._value = internString(value);
```

only stored values should be interned

Reply:
Done.